db/migrations/20221101130018_add_display_meta_data_migration.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
Needs some copyright info here - please set up the copyright profile for your IDE so its automatically added/updated